pyvex/lifting/util/lifter_helper.py

```diff
@@ -86,21 +103,47 @@ def lift(self, disassemble=False, dump_irsb=False):
 
         if disassemble:
             return [instr.disassemble() for instr in instructions]
-        self.irsb.jumpkind = JumpKind.Invalid
-        irsb_c = IRSBCustomizer(self.irsb)
+
         l.debug("Decoding complete.")
+        self.irsb.jumpkind = None
         for i, instr in enumerate(instructions[:self.max_inst]):
+            # first, create a scratch IRSB so we can throw the changes away if lifting fails
+            next_irsb_part = pyvex.IRSB.empty_block(self.irsb.arch, self.irsb.addr)
```

Comment:
`pyvex` is no longer imported. In fact, using `pyvex` inside the PyVEX package is not a good choice...